core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -695,12 +707,21 @@ object Stream {
 
   implicit def covaryPurePipe2[F[_],I,I2,O](p: Pipe2[Pure,I,I2,O]): Pipe2[F,I,I2,O] = pipe2.covary[F,I,I2,O](p)
 
-  implicit def streamCatchableInstance[F[_]]: Catchable[({ type λ[a] = Stream[F, a] })#λ] =
-    new Catchable[({ type λ[a] = Stream[F, a] })#λ] {
+  implicit def streamMonadErrorInstance[F[_]]: MonadError[Stream[F,?], Throwable] =
```

Comment:
Random thought: shouldn't `Stream[F, ?]` implement `Concurrent` for all `Concurrent[F]`?  By the same token, it could implement `Effect` for all `Effect[F]`, etc up the line.